AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRColorGradingAdded.py

```diff
@@ -146,42 +304,427 @@ def AtomEditorComponents_HDRColorGrading_AddedToEntity():
+        # Set Post Saturation to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Post Saturation'), -100.0)
+        Report.result(Tests.post_saturation_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Post Saturation')) == -100.0)
+
+        # Set Post Saturation to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Post Saturation'), 100.0)
+        Report.result(Tests.post_saturation_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Post Saturation')) == 100.0)
+
+        # 37. Hue Shift (float range 0.0 to 1.0, default 0.0)
+        # Set Hue Shift to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Hue Shift'), 1.0)
+        Report.result(Tests.hue_shift_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Hue Shift')) == 1.0)
+
+        # Cycle through LUT Resolutions.
+        for lut_resolution in LUT_RESOLUTION.keys():
+
+            # 38. LUT Resolution (from atom_constants.py LUT_RESOLUTION, default 16x16x16)
+            hdr_color_grading_component.set_component_property_value(
+                AtomComponentProperties.hdr_color_grading('LUT Resolution'), LUT_RESOLUTION[lut_resolution])
+            test_lut_resolution = (
+                f"Set Lut Resolution to: {lut_resolution}",
+                f"P1: Lut Resolution failed to be set to {lut_resolution} ")
+            Report.result(test_lut_resolution, hdr_color_grading_component.get_component_property_value(
+                AtomComponentProperties.hdr_color_grading('LUT Resolution')) == LUT_RESOLUTION[lut_resolution])
+
+            # Cycle through Shaper Types per LUT Resolution.
+            for shaper_type in SHAPER_TYPE.keys():
+
+                # 39. Set the Shaper Type parameter (from atom_constants.py SHAPER_TYPE, default none)
+                hdr_color_grading_component.set_component_property_value(
+                    AtomComponentProperties.hdr_color_grading('Shaper Type'), SHAPER_TYPE[shaper_type])
+                test_shaper_type = (
+                    f"Set Shaper Type to: {shaper_type}",
+                    f"P1: Shaper Type failed to be set to {shaper_type} ")
+                Report.result(test_shaper_type, hdr_color_grading_component.get_component_property_value(
+                    AtomComponentProperties.hdr_color_grading('Shaper Type')) == SHAPER_TYPE[shaper_type])
+
+        # 42. Enter/Exit game mode.
```

Comment:
should be # 40.

Reply:
Fixed that (and all subsequent numbering) on both this and the docstring.